src/ccnl-core/src/ccnl-prefix.c

```diff
@@ -595,29 +595,29 @@ ccnl_prefix_to_str_detailed(struct ccnl_prefix_s *pr, int ccntlv_skip, int escap
     (void)j;
     (void)len;
     (void) call_slash;
-    (void)buflen; //fixme, prevent bufferoverrun here!
+    (void)buflen;
 #ifdef USE_NFN
-    // len += sprintf(buf + len, "cmpcnt: %i|", pr->compcnt);
+    // len += snprintf(buf + len, buflen - len, "cmpcnt: %i|", pr->compcnt);
     if (pr->nfnflags & CCNL_PREFIX_NFN) {
-        len += sprintf(buf + len, "nfn");
+        len += snprintf(buf + len, buflen - len, "nfn");
     }
 #ifdef USE_NFN_REQUESTS
-    // len += sprintf(buf + len, ":test");
+    // len += snprintf(buf + len, buflen - len, ":test");
     if (pr->nfnflags & CCNL_PREFIX_REQUEST) {
         char *desc = nfn_request_description_new(pr->request);
-        len += sprintf(buf + len, ":%s", desc);
+        len += snprintf(buf + len, buflen - len, ":%s", desc);
         ccnl_free(desc);
         // if (pr->request == NULL) {
-        //     len += sprintf(buf + len, ":request(?)");
+        //     len += snprintf(buf + len, buflen - len, ":request(?)");
         // } else {
-        //     len += sprintf(buf + len, ":request(%d)", pr->request->complen);
+        //     len += snprintf(buf + len, buflen - len, ":request(%d)", pr->request->complen);
         // }
-        // len += sprintf(buf + len, ":request(%c)", *(pr->request->comp));
-        // len += sprintf(buf + len, ":request(%.*s)", pr->request->complen, pr->request->comp);
+        // len += snprintf(buf + len, ":request(%c)", *(pr->request->comp));
```

Comment:
What is it with the code which is commented out? Can we remove it? This applies to all of the commented out code in this PR.

Reply:
Will do that, even if I do not exactly understand for that this code was written in first place

Reply:
Don't get me wrong. If you think there is some kind of educational purpose leave it. I honestly can't make anything out of it and I'm wondering if anybody does.

Reply:
No, I think somebody used that for debugging. I think we are good to remove that